src/Integration.Vsix/SonarLintDaemon/Protocol.cs

```diff
@@ -0,0 +1,231 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace SonarLint.VisualStudio.Integration.Vsix.SonarLintDaemon
```

Comment:
I was wondering whether to put the new C++ integration code in a separate folder/namespace to make it easy to see which bits are for the Java daemon and which bits are C++.
For now, I think it would be a good idea to put this class in a "cpp" subfolder.